public/components/feature-switch/feature-switch.html

```diff
@@ -0,0 +1,7 @@
+<button class="feature-switch-toggle" ng-click="toggleFeatureSwitch()" title="Toggle key takeaways creation">
+  Toggle key takeaways creation
+  <div
+    class="feature-switch-toggle__icon"
+    ng-class="{'feature-switch-toggle__icon--active': featureSwitch.visible }"
+  ></div>
+</button>
```

Comment:
this could be modelled as a checkbox with the label as just `Key Takeaways creation`

or failing that, you could at least alter the text based on current value, something like this
```suggestion
<button class="feature-switch-toggle" ng-click="toggleFeatureSwitch()" title="Toggle key takeaways creation">
 {featureSwitch ? 'ENABLE' : 'DISABLE'} Key Takeaways creation
  <div
    class="feature-switch-toggle__icon"
    ng-class="{'feature-switch-toggle__icon--active': featureSwitch.visible }"
  ></div>
</button>
```